static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -72,35 +76,54 @@ define([
-                quoteText: 'Support and become part of the Guardian',
-                quoteAuthor: 'Hugh Muir',
-                customCssClass: 'muir',
-                imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/pictures/2014/3/13/1394733739000/HughMuir.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=c1eeb35230ad2a215ec9de76b3eb69fb'
+            contributorBannersUK = [
+                {
+                    edition: 'UK',
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_MONBIOT',
+                    quoteText: 'Become a Guardian Member and support independent journalism',
+                    quoteAuthor: 'George Monbiot',
+                    customCssClass: 'monbiot',
+                    imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2015/7/9/1436429159376/George-Monbiot-L.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=903233b032379d7529d7337b8c26bcc9'
+                },
+                {
+                    edition: 'UK',
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_MUIR',
+                    quoteText: 'Support and become part of the Guardian',
+                    quoteAuthor: 'Hugh Muir',
+                    customCssClass: 'muir',
+                    imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/pictures/2014/3/13/1394733739000/HughMuir.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=c1eeb35230ad2a215ec9de76b3eb69fb'
+                },
+                {
+                    edition: 'UK',
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_TOYNBEE',
+                    quoteText: 'If you read the Guardian, join the Guardian',
+                    quoteAuthor: 'Polly Toynbee',
+                    customCssClass: 'toynbee',
+                    imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2014/6/30/1404146756739/Polly-Toynbee-L.png?w=300&amp;q=85&amp;auto=format&amp;sharp=10&amp;s=abf0ce1a1a7935e82612b330322f5fa4'
+                },
+                {
+                    edition: 'UK',
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_MACASKILL',
+                    quoteText: 'The Guardian enjoys rare freedom and independence. Support our journalism',
+                    quoteAuthor: 'Ewen MacAskill',
+                    customCssClass: 'macaskill',
+                    imageAuthor: '//i.guim.co.uk/img/static/sys-images/Guardian/Pix/contributor/2015/8/18/1439913873894/Ewen-MacAskill-R.png?w=300&q=85&auto=format&sharp=10&s=0ecfbc78dc606a01c0a9b04bd5ac7a82'
+                }
+            ],
+
+            internationaliseBanner = function (country) {
+                return function (contributor) {
```

Comment:
This seems an odd signature to me - I would make this perform the merge directly. If you need to curry functions with parameters when passing it to a `map` (as you seem to be later) you can use Lodash's `_.partial` helper.